content/en/hosting/costs/_index.md

```diff
@@ -0,0 +1,95 @@
+* Ingress/Egress - This is the ability to send and receive data to the CHT.  Pricing from Datacenters and Cloud is readily available and "Servers" above can not exist without it.
+* CHT Core updates and system maintenance - Both the CHT and underlying operating system will be upgraded on a regular basis.  This not only ensures there are no security vulnerabilities, but also ensures the deployments gains the benefit of new features and performance gains in the CHT.
+
+### Excluded
+
+There are many assumptions about what else is needed to run the CHT. While important, they are excluded:
+
+* Kubernetes cluster - Current CHT production best practices include a Kubernetes cluster that will have a fixed monthly cost to operate, including the worker nodes which may be large in size (RAM & CPU).
+* Backups - Regular snapshots of the production data will need to be taken to ensure there is no data loss in case of catastrophic failure of server hardware.  This takes up a disk space which should be accounted for when budgeting to host the CHT.
+* Building a Datacenter (DC) - While some well funded MoH's may have the budget and time to build a complete data center, this is out of the scope of Medic's core competency.  Therefore it is assumed an existing DC will be used or cloud hosting will be used.  In the case where a MoH wants to build a DC, a competent 3rd party should be selected
+* Training of Systems Administrators/IT - System administrator IT systems that have not used Kubernetes or have not hosted the CHT, will need to be trained on how to do both.
+* Training of Trainers (ToT) and user Training -
+* Data Warehousing & Dashboards - While close to "Servers" above, not all deployments have a data warehouse (CHT Sync + Postgres) and dashboards (Superset/Klipfolio) at launch.  These are easy to add on at a later date and cost can be estimated at that time.
+* Upfront Purchase of Hardware - It is assumed that a deployment will either be using cloud based solutions or using managed bare metal, so these costs don't apply.
+* App Development - Each deployment needs to have the default CHT app customized for each of the required workflows.
+* Smartphones - Device purchase, setup, and distribution for users to use the CHT.
+* Analog -> Digital Workflow conversion - The process of documenting paper processes in places that the CHT will replace.
+* SMS - Some projects need the ability to send SMS to users from the CHT.
+* Interoperability - While the CHT supports this out of the box - development work is needed to ensure it works with the specific 3rd party systems.
+* Mobile Device Management - optional.
+* Cellphone telecom services - Users actually uploading/downloading data
+* Transfer of hosting from Medic hosted to MoH Self Hosted - Many person hours needed to help with transfer of data to new MoH based hosting.
```

Comment:
Recommend this is removed as it's a one off. Or possibly changed to talk about server set up rather than calling out Medic specifically. eg:

```suggestion
* Setup and/or transfer of hosting provider
```